action.yml

```diff
@@ -233,6 +238,7 @@ runs:
         exit $rc
       shell: bash
     - name: Copy artifacts within image
+      if: ${{ always() && inputs.copy_artifacts_on_fail == 'yes' || steps.runcmd.conclusion != 'failure' }}
```

Comment:
Could we please use `success()` instead of `always()` here? I believe `always()` has a special semantic regarding cancelled tasks.

Reply:
perhaps a combination of `always() && !cancelled()` in this way it should skip the step if the job get canceled